### PR TITLE
Doc mobile styles

### PR DESCRIPTION
**Pull request by lydialowther**
Both the Menu and TOC need to be hooked up with Javascript. (Yes, I still need to find a TOC logo)

screen shot 2016-08-31 at 3 34 38 pm
screen shot 2016-08-31 at 3 35 43 pm

### CODE REVIEW

**Comment by bryanp** [comment]
Very nice @lydialowther! I had to stop what I was doing when I saw the email just to say that. The screenshots are superb. Will take a closer look when I can 🙂

**Comment by lydialowther** [comment]
screen shot 2016-09-20 at 2 05 23 pm

File: app/views/docs/_header.slim [hunk 1]
```diff
@@ -7,21 +7,29 @@ header
...
-      | Docs
-      
-    a@item-forums href="http://forums.pakyow.org"
-      | Forums
-
-    a@item-code href="https://github.com/pakyow/pakyow"
-      i.fa.fa-github-square
-      | Code
+    .nav-items
+      ul
```
Comment (bryanp) [inline app/views/docs/_header.slim]: Let's move `.nav-items` to be on the `ul` element.

**Comment by bryanp** [comment]
@lydialowther I just pushed a commit that hooks up the nav and cleans up the styles. Give it a look and let me know what you think.

**Comment by lydialowther** [comment]
I think It looks good. I might increase the font size of the link that pulls out the topic list. Also you might be able to get away with just "topics" and an X to close the list.

**Comment by bryanp** [comment]
Awesome. Gonna just merge and go for it. Tired of tweaking this one :)

**Pull request merged**